src/languages/sass.coffee

```diff
@@ -2,7 +2,6 @@ module.exports = {
 
   name: "Sass"
   namespace: "sass"
-  fallback: ['css', 'scss']
```

Comment:
The `fallback` field is a great design. I deleted this line simply because Sass has no fallbacks. It's another way to write CSS and could be compiled into CSS. however, it's completely different from CSS (no braces, indents as scopes, etc).

I don't think any CSS beautifier could handle Sass well. Take #1088 as an example.